bands/templates/band_admin/guest_info.html

```diff
@@ -0,0 +1,91 @@
+<style type="text/css">
+    .form-group { margin-bottom: 8px; }
+</style>
+
+<h2>Guest Info for <a href="../bands/index?id={{ band.id }}">{{ band.group.name }}</a></h2>
```

Comment:
Most of this is copy-pasted from the Band Info plugin.. it was not clear to me what the guest info section should include, so I decided not to change much yet. This is more for allowing full customization of the admin view later on.